src/installing-workflow/experimental-native-ingress.md

```diff
@@ -0,0 +1,80 @@
+# Experimental Native Ingress
+
+## Install Deis Workflow (With experimental native ingress support)
+
+Now that Helm is installed and the repository has been added, install Workflow with a native ingress by running:
+
+```
+$ helm install deis/workflow --namespace deis --set global.experimental_native_ingress=true,controller.platform_domain=deis.com
+```
+
+Where `global.hostname` is a **required** parameter that is traditionally not required for Workflow. In this example we are using `deis.com` for `$hostname`.
+
+
+ 
+Helm will install a variety of Kubernetes resources in the `deis` namespace.
+Wait for the pods that Helm launched to be ready. Monitor their status by running:
+
+```
+$ kubectl --namespace=deis get pods
+```
+
+You should also notice that a Kubernetes ingress has been installed on your cluster. You can view it by running:
```

Comment:
"ingress" is the proper terminology here, as @kris-nova is referring to ingress _resources_ that are part of the chart and define routing rules for the controller, grafana, etc.

That being said, "a Kubernetes ingress," might need to change because I do think there's more than one installed as part of the platform itself.